libpod/container_api.go

```diff
@@ -715,6 +722,12 @@ func (c *Container) Checkpoint(ctx context.Context, options ContainerCheckpointO
 		}
 	}
 
+	if options.WithPrevious {
```

Comment:
I think you want a check to make sure WithPrevious and pre-checkpoint are not used at the same time.

Reply:
Should I add it in cmd/podman/containers/checkpoints.go, or libpod/container_api.go?

Reply:
My thought was here, but where ever you feel is best.